packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -0,0 +1,251 @@
+import flatten from "flat";
+import React, { ReactElement } from "react";
+import { IntlProvider } from "react-intl";
+import { Story as StoryType } from "@storybook/react";
+import styled from "styled-components";
+// The below eslint-disable is due to https://github.com/storybookjs/storybook/issues/13408
+// eslint-disable-next-line import/no-named-as-default
+import TripDetails from ".";
+import * as TripDetailsClasses from "./styled";
+import {
+  CaloriesDetailsProps,
+  DepartureDetailsProps,
+  FareDetailsProps
+} from "./types";
+
+import defaultEnglishMessages from "../i18n/en-US.yml";
+import defaultFrenchMessages from "../i18n/fr.yml";
```

Comment:
To reduce confusion with default messages for react-intl, I'd simply call these as follows:

```suggestion
import englishMessages from "../i18n/en-US.yml";
import frenchMessages from "../i18n/fr.yml";
```